plugins/outputs/azure_data_explorer/azure_data_explorer.go

```diff
@@ -209,14 +201,18 @@ func (adx *AzureDataExplorer) createAzureDataExplorerTable(ctx context.Context,
 		adx.Log.Info("skipped table creation")
 		return nil
 	}
-	createStmt := kusto.NewStmt("", kusto.UnsafeStmt(unsafe.Stmt{Add: true, SuppressWarning: true})).UnsafeAdd(fmt.Sprintf(createTableCommand, tableName))
-	if _, err := adx.kustoClient.Mgmt(ctx, adx.Database, createStmt); err != nil {
+
+	createStmt := kql.New(createTableCommand)
+	createParams := kusto.QueryParameters(kql.NewParameters().AddString("TABLE", tableName))
+	if _, err := adx.kustoClient.Mgmt(ctx, adx.Database, createStmt, createParams); err != nil {
 		return err
 	}
 
-	createTableMappingstmt := kusto.NewStmt("", kusto.UnsafeStmt(unsafe.Stmt{Add: true, SuppressWarning: true})).
-		UnsafeAdd(fmt.Sprintf(createTableMappingCommand, tableName, tableName))
-	if _, err := adx.kustoClient.Mgmt(ctx, adx.Database, createTableMappingstmt); err != nil {
+	createTableMappingStmt := kql.New(createTableMappingCommand)
+	createTableMappingParams := kusto.QueryParameters(
+		kql.NewParameters().AddString("TABLE", tableName).AddString("TABLEMAPPING", tableName+"_mapping"),
```

Comment:
does addstring match TABLE and only TABLE? What happens if the query, while unlikely, happened to have TABLE in it?